ci.hocon

```diff
@@ -531,10 +531,12 @@ builds: [
   {name: ruby-metrics-compiler-graal-core} ${common} ${graal-core} ${bench-caps} ${truffleruby} ${compiler-metrics},
   {name: ruby-metrics-compiler-graal-enterprise} ${common} ${graal-enterprise} ${daily-bench-caps} ${truffleruby} ${compiler-metrics},
   {name: ruby-metrics-compiler-graal-enterprise-no-om} ${common} ${graal-enterprise-no-om} ${daily-bench-caps} ${truffleruby} ${compiler-metrics},
-  //{name: ruby-metrics-compiler-graal-vm-snapshot} ${common} ${graal-vm-snapshot} ${bench-caps} ${truffleruby} ${compiler-metrics},
-  {name: ruby-build-stats-svm} ${common} ${svm-bench} ${bench-caps} ${truffleruby} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-build-stats},
-  {name: ruby-metrics-svm-with-rubygems} ${common} ${svm-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-metrics},
-  {name: ruby-metrics-svm-without-rubygems} ${common} ${svm-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: no-rubygems, TRUFFLERUBYOPT: ''}} ${svm-metrics},
+  {name: ruby-build-stats-svm-graal-core} ${common} ${svm-graal-core-bench} ${bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-build-stats},
+  {name: ruby-build-stats-svm-graal-enterprise} ${common} ${svm-graal-enterprise-bench} ${bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-build-stats},
+  {name: ruby-metrics-svm-graal-core-with-rubygems} ${common} ${svm-graal-core-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-metrics},
+  {name: ruby-metrics-svm-graal-enterprise-with-rubygems} ${common} ${svm-graal-enterprise-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: default}} ${svm-metrics},
+  {name: ruby-metrics-svm-graal-core-without-rubygems} ${common} ${svm-graal-core-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: no-rubygems, TRUFFLERUBYOPT: ''}} ${svm-metrics},
+  {name: ruby-metrics-svm-graal-enterprise-without-rubygems} ${common} ${svm-graal-enterprise-bench} ${svm-bench-caps} ${truffleruby} {environment: {GUEST_VM_CONFIG: no-rubygems, TRUFFLERUBYOPT: ''}} ${svm-metrics},
```

Comment:
If you know how to do it cleanly, I'm happy to make the change. I don't have the desire to spend another 3 days debugging environment variable overrides and what not to combine these right at the moment. If it's really problematic, I'll just comment out the ones we didn't have before.